Source/WebCore/css/values/color/CSSColorMix.cpp

```diff
@@ -90,9 +90,95 @@ bool containsColorSchemeDependentColor(const ColorMix& unresolved)
         || containsColorSchemeDependentColor(unresolved.mixComponents2.color);
 }
 
+namespace ColorMixSerializationDetails {
+
+static bool is50Percent(const ColorMix::Component::Percentage& percentage)
+{
+    return WTF::switchOn(percentage,
+        [](const ColorMix::Component::Percentage::Raw& raw) { return raw.value == 50.0; },
+        [](const ColorMix::Component::Percentage::Calc&) { return false; }
+    );
```

Comment:
Given the style of the function just below, maybe this instead?
```suggestion
    return !percentage.isCalc() && percentage.raw()->value == 50.0;
```
Or maybe we want to use `switchOn` except when there are multiple objects involved?

Reply:
I can actually do this one so much better.

```
return percentage == 50_css_percentage;
```

Boom.